src/couch_index/test/eunit/couch_index_crash_tests.erl

```diff
@@ -78,6 +78,21 @@ index_crash_test_() ->
         }
     }.
 
+index_crash_test_with_client_waiting_test_() ->
```

Comment:
This would probably still be a good name without the first `_test_`.

Reply:
Oh I was silly, it was a typo after a copy/paste/rename.

Thanks for noticing, Jay.